### PR TITLE
Stripe widget

### PR DESCRIPTION
**Pull request by richardcolachagua**
Stripe Widget UI first draft. Ignore the commit from the contract status widget. I forgot to make a separate branch before committing.

### CODE REVIEW

File: src/components/ContractStatusWidget/ContractStatus.js
Comment (AlanisYates) [inline src/components/ContractStatusWidget/ContractStatus.js:1]: Remove from PR

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx
Comment (AlanisYates) [inline src/components/ContractStatusWidget/ContractStatusWidget.jsx:1]: Same remove from this PR

**Review by AlanisYates — CHANGES_REQUESTED** [review]

**Review by richardcolachagua — COMMENTED** [review]
Removed contract files

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx
Comment (richardcolachagua) [reply src/components/ContractStatusWidget/ContractStatusWidget.jsx:1]: removed

File: src/components/ContractStatusWidget/ContractStatus.js
Comment (richardcolachagua) [reply src/components/ContractStatusWidget/ContractStatus.js:1]: removed

**Review by richardcolachagua — COMMENTED** [review]
removed files

**Review by AlanisYates — CHANGES_REQUESTED** [review]
Remove the contract status files

**Comment by AlanisYates** [comment]
Also pull in main

**Review by richardcolachagua — COMMENTED** [review]
"made contract status changes and pulled latest"

**Pull request closed without merging**